spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/topic-resolution.adoc

```diff
@@ -48,6 +48,20 @@ record Foo(String value) {
 }
 ----
 
+Property placeholders and SpEL expressions are supported in the `@PulsarMessage` annotation,
+for example:
+[source,java,indent=0,subs="verbatim"]
+----
+@PulsarMessage(topic = "${app.topics.foo}")
+record Foo(String value) {
+}
+----
+
+WARNING: Property placeholders and SpEL expressions are by default supported within a Spring Boot context.
+
+NOTE: If you are using the framework outside a Spring Boot context, you will need to construct a `PulsarTemplate`
```

Comment:
Based on above comment this one section can be removed. Also, topic-resolution is shared by template and listener so it has to be template and listener agnostic.